app/admin/system/infos.rb

```diff
@@ -32,6 +32,15 @@
           end
         end
 
+        if YetiConfig.show_config_info.present?
+          panel 'Config' do
+            attributes_table_for(YetiConfig) do
+              row :keep_expired_destinations_days
+              row :keep_expired_dialpeers_days
+            end
+          end
+        end
```

Comment:
1. rename to `Yeti-web config`

2. add the following rows:
nested hash can be shown as lines "key: value"
nested array can be shown as lines "value"
```
calls_monitoring:
  write_account_stats: true
  write_gateway_stats: true
api:
  token_lifetime: 600 # jwt token lifetime in seconds, empty string means permanent tokens
cdr_export:
  dir_path: "/tmp"
  delete_url: http://localhost/cdrs_export/

role_policy:
  when_no_config: disallow
  when_no_policy_class: raise

partition_remove_delay:
  cdr.cdr: 120 days
  auth_log.auth_log: 7 days
  rtp_statistics.rx_streams: 3 days
  rtp_statistics.tx_streams: 3 days
  logs.api_requests: 90 days

prometheus:
  enabled: false
  default_labels:
  host: my-host

sentry:
  enabled: false
  node_name: "local"
  environment: "prod"

versioning_disable_for_models:
  - ClassName::ClassName
  - ClassName
```